apps/client/src/externals.ts

```diff
@@ -17,11 +17,11 @@ export const buyMeACoffeeUrl = 'https://buymeacoffee.com/cpvalente';
 
 // resolve environment
 export const appVersion = version;
-export const isProduction = import.meta.env.MODE === 'production';
-export const isDev = !isProduction;
+export const isDocker = import.meta.env.IS_DOCKER;
```

Comment:
question: do we no longer need to prepend the env variables with `VITE_`?
https://vite.dev/guide/env-and-mode.html#env-variables

Reply:
a-ha, thank you. that makes sense.
@alex-Arc In that case I think we could do with a small comment in the vite.config.js.

I believe that this creates an indirection which is not particularly explicit and can cause confusion in future